src/main/java/com/opentok/Archive.java

```diff
@@ -252,6 +257,13 @@ public String getPassword() {
         return password;
     }
 
+    /**
+     * Returns the multiArchiveTag if set for the Archive.
+     */
+    public String getMultiArchiveTag() {
+        return multiArchiveTag;
+    }
+
```

Comment:
I don't think we should have `Archive. getMultiArchiveTag()` and `Broadcast.getMultiBroadcastTag()` methods. The responses for `get archive` and `get broadcast` methods do not return the `multi...tag` properties. (Please confirm.)

Reply:
Seems like you're right according to the specs. It was the presentation that confused me, since it shows these properties in the responses. Presumably the presentation predates the current spec.